src/main/webapp/app/instructor-course-dashboard/tutor-leaderboard/tutor-leaderboard.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="row table-responsive" style="margin-top: 15px">
+    <table class="table">
+        <thead>
+        <tr>
+            <th>ID</th>
+            <th>Name</th>
+            <th>Username</th>
+            <th>Nr. of assessments</th>
+            <th>Nr. of complaints</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr *ngFor="let tutor of tutorsData | keyvalue">
```

Comment:
keyvalue necessary?

Reply:
Yes, because tutorsData is an object and we need an array: https://angular.io/api/common/KeyValuePipe